README.md

```diff
@@ -1,41 +1,196 @@
+
+There are also files for the standard ssh keys and definitions for the wifi profiles.
+
+#### model_files
+
+These files define how bbb-configs needs to handle different hardware models. This example shows a WDR4900:
+```sh
+---
+imagebuilder: "https://downloads.openwrt.org/releases/21.02-SNAPSHOT/targets/mpc85xx/p1010/openwrt-imagebuilder-21.02-SNAPSHOT-mpc85xx-p1010.Linux-x86_64.tar.xz"
+feed_version: "1.2.0-snapshot"
+imagebuilder_disable_signature_check: true      # ignore missing signatures for packages (i.e. selfcompiled)
+
+switch_ports: 6             # number of physical ports + one (CPU)
+switch_int_port: 0          # port-id of the CPU
+switch_ignore_ports: []     # omit ports, that exist in software but not in hardware (i.e. MikroTik SXTsq 5ac)
+
+int_port: eth0              # hardware-device on which swconfig works on
+
+wireless_devices:                       # definitions for the devices radios
+  - name: 11a_standard                  # 5GHz radio
+    hwmode: 11a
+    path: ffe09000.pcie/pci9000:00/9000:00:00.0/9000:01:00.0
+    ifname_hint: wlan5
+  - name: 11g_standard                  # 2.4GHz radio
+    hwmode: 11g
+    path: ffe0a000.pcie/pcia000:02/a000:02:00.0/a000:03:00.0
+    ifname_hint: wlan2
+```
+
+For a model using DSA instead of swconfig, you may refer to [`model_ubnt_edgerouter_x_sfp.yml`](https://github.com/Freifunk-Spalter/bbb-configs/blob/master/group_vars/model_ubnt_edgerouter_x_sfp.yml)
+
+#### location-directories
+
+When defining a new location, you need to create a directory named like `location_$NAME/`. In that directory goes the location specific configuration. It consists of several files. You can override the variables defined from the `all` directory by creating a file named like that one in `all/` and redefine any variable.
+
+##### general.yml
+
+This file might be fairly self-explanatory. Please mind that a contact is mandatory. If you don't like to give your email address, you can use the link to the contact form, that you've got from [config.berlin.freifunk.net](https://config.berlin.freifunk.net)
+
+```sh
+---
+contact_nickname: 'Petrosilius Quaccus'
+contact_email: 'quaccus@example.com'
 ```
-/group_vars : Create location folder
-/host_vars : Create host folder for every openwrt device
-Done!
 
+##### owm.yml
+
+This file defines some values to get the location to the [OpenWifiMap](https://openwifimap.net).
+
+```sh
+---
+location_nice: roof top, Example-Street 24, 10573 Berlin        # any string that describes your location. Could be an address or something
+latitude: 52.484948320
+longitude: 13.443380903
+```
+
+##### networks.yml
+
+This file is the most important one of your setup. It defines IP-Addresses, WIFI-Properties, Mesh and so on. Lets have a (shortened) example of a `networks.yml` from the magda-location:
+
+```sh
```

Comment:
That should be yaml i guess. I never use that infostring, not sure if you did this on purpose

Reply:
I did it on purpose.vscode complained, that markdown standard requires that.